pkg/man/sparkRSQL.init.Rd

```diff
@@ -1,7 +1,7 @@
 % Generated by roxygen2 (4.0.2): do not edit by hand
 \name{sparkRSQL.init}
 \alias{sparkRSQL.init}
-\title{Initialize a new Spark Context.}
+\title{Initialize a new SQLContext.}
```

Comment:
Yeah, so far we have kept the generated docs in git as we don't have a nice way to get it into the installed package. We can revisit this when we try to merge with mainline Spark.